src/d3a/models/market/market_redis_connection.py

```diff
@@ -265,7 +265,7 @@ def _accept_bid_impl(self, arguments):
                          {"status": "ready", "trade": trade.to_JSON_string(),
                           "transaction_uuid": transaction_uuid})
         except Exception as e:
-            logging.error(f"Error when handling accept_bid on market {self.market_object.name}: "
+            logging.error(f"Error when handling accept_bid on market {self.market.name}: "
```

Comment:
Upsi. This got throught the last time, right?

Reply:
No, we were obliged to revert this comment actually. The reason for that is because there needs to be a property in order to get overriden, which conflicts to the 'market' instance variable name, thus causing errors.